inference-engine/tests/functional/inference_engine/serialization/single_layer/prior_box_clustered.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "shared_test_classes/single_layer/prior_box_clustered.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+    TEST_P(PriorBoxClusteredLayerTest, Serialize) {
+        Serialize();
+    }
+
+    const std::vector<InferenceEngine::Precision> netPrecisions = {
```

Comment:
The three declarations starting here (netPrecisions, widths, heights) have different indentation